src/screens/PickList/index.tsx

```diff
@@ -1,19 +1,12 @@
 import React, {useEffect, useState} from 'react';
-import {DispatchProps} from './types';
 import styles from './styles';
 import {ListRenderItemInfo, ScrollView, Text, View} from 'react-native';
 import {pickListVMMapper} from './PickListVMMapper';
-import {hideScreenLoading, showScreenLoading} from '../../redux/actions/main';
-import {connect, useDispatch} from 'react-redux';
+import {hideScreenLoading} from '../../redux/actions/main';
```

Comment:
Please bring back previous version of the imports

Reply:
I have removed Unused import only why do we need that